python/pybind11/README.rst

```diff
@@ -0,0 +1,183 @@
+Note: This is a dump of the pybind11 distribution, version 2.11.0.
```

Comment:
This line was added to the original file in https://github.com/pybind/pybind11/archive/refs/tags/v2.11.0.tar.gz in accordance with the comment https://github.com/OPM/opm-common/pull/819#issuecomment-502545455